src/main/java/thePackmaster/powers/dimensiongate3pack/MantisStrikePower.java

```diff
@@ -0,0 +1,51 @@
+package thePackmaster.powers.dimensiongate3pack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.NonStackablePower;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.AttackDamageRandomEnemyAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class MantisStrikePower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("MantisStrikePower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public MantisStrikePower(AbstractCreature owner, int damageAmount) {
+        super(POWER_ID, NAME, PowerType.BUFF, true, owner, 1);
```

Comment:
It looks like `1` is hardcoded as the amount being passed to the superclass constructor here, which I think is causing a bug where Mantris Strike's buff does the wrong damage. Also, this shouldn't be marked as a turn-based power.
```suggestion
        super(POWER_ID, NAME, PowerType.BUFF, false, owner, damageAmount);
```

Once you make this change you can delete the `amount = damageAmount` and `updateDescription()` lines below, the superclass constructor takes care of all that.